webapp/channels/.stylelintrc.json

```
@@ -22,6 +22,19 @@
                 "ignore": ["comments"]
             }
         ],
+        "declaration-property-value-disallowed-list": [
+            {
+                "color": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"],
+                "background-color": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"],
+                "border-color": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"],
+                "background": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"],
+                "border": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"],
+                "fill": ["var(--denim-button-bg)", "var(--denim-button-bg-rgb)"]
```

This just prevents those settings from being set to those exact values, but that doesn't work when we use the rgb versions or for rules like `background` and `border` where you pass multiple values at once. This should cover all of those cases though by making them into regular expressions that check if the value contains `--denim-button-bg` (or `--denim-buton-bg-rgb`) at all

```suggestion
                "color": ["/--denim-button-bg/"],
                "background-color": ["/--denim-button-bg/"],
                "border-color": ["/--denim-button-bg/"],
                "background": ["/--denim-button-bg/"],
                "border": ["/--denim-button-bg/"],
                "fill": ["/--denim-button-bg/"]
```